src/types/ErrorRate.ts

```diff
@@ -0,0 +1,171 @@
+import { NA, RequestHealth } from './Health';
+import { RateHealthConfig, ToleranceConfig } from './ServerConfig';
+import { serverConfig } from '../config/ServerConfig';
+import { ThresholdStatus, getRequestErrorsStatus, RATIO_NA } from './Health';
+
+export interface ErrorRatio {
```

Comment:
to avoid the repetition (like above: `x.errorRatio.errorRatio.status`) we could rename and shorten these fields to `global`, `inbound` and `outbound`?